CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
```

Comment:
Could you take a loom of testing out the code and generating a nuxt js app?

Reply:
Please also include a call to the sessioninfo endpoint with the session tokens in the demo. Thanks

Reply:
Here it is
https://jam.dev/c/a3625633-1562-4d7d-a5b5-895492caa27e

Reply:
Im getting the following error when i try and call the sign in API:
```
Unexpected token o in JSON at position 1

  at JSON.parse (<anonymous>)
  at vr.json (node_modules/node-fetch-native/dist/node.mjs:17:2658)
  at processTicksAndRejections (node:internal/process/task_queues:96:5)
  at Object.getJSONBody (server/superTokensHelper.ts:121:20)
  at async PreParsedRequest.BaseRequest.getJSONBody (node_modules/supertokens-node/lib/build/framework/request.js:40:39)
  at async Object.signInAPI [as default] (node_modules/supertokens-node/lib/build/recipe/emailpassword/api/signin.js:33:10)
  at async Recipe.handleAPIRequest (node_modules/supertokens-node/lib/build/recipe/emailpassword/recipe.js:103:24)
  at async SuperTokens.middleware (node_modules/supertokens-node/lib/build/supertokens.js:256:38)
  at async node_modules/supertokens-node/lib/build/framework/custom/framework.js:133:28
  at handleCall (server/superTokensHelper.ts:58:36)
```

Reply:
Would you please elaborate, I am not getting any error. 
make sure you have 
- [ ] generated nuxt js app using `npx . --branch=feat/nuxt-prebuilt`
- [ ] `cd <directory>` `npm install` and `npm run dev`. Make sure it's open on port `3000`.

for me signin is working fine. 

If this doesn't resolve the issue, please share a loom.

Reply:
is this issue still exist?

Reply:
@deepjyoti30-st from our team will try it out and get back to you.

Reply:
Hi @deepjyoti30-st please let me know if it is working